docs/using-est-with-certd.md

```diff
@@ -0,0 +1,169 @@
+# Using EST with Certificates Service
+
+This document covers how to configure Certificates Service to issue certificates over EST.
+
+## Prerequisites
+
+- Certificates Service and Keys Service configured and running
+- A working EST server that processes enrollment requests
+
+## Configuration Options
+
+EST certificate issuance is configured in Certificates Service's config.toml, which by default is `/etc/aziot/certd/config.toml` or a file in `/etc/aziot/certd/config.d/`.
```

Comment:
Why tell people to do it in certd config instead of super-config?

Reply:
This doc about issuing an arbitrary certificate over EST. The super config only supports issuing the device ID and Edge CA certs over EST (not any arbitrary cert).

I'll be writing another doc that covers the device ID and Edge CA certs specifically. That will use the super config.